docs/sources/metainfo/component.xml

```diff
@@ -253,17 +253,56 @@
-		</para>
-		<para>
-			The <code>&lt;launchable/&gt;</code> tag has a essential <literal>type</literal> property indicating the system that is used to launch the application. At time, only
-			<code>desktop-id</code> is allowed as value, indicating that an application can be launched via a desktop file. The value of the tag is a
-			<ulink url="https://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#desktop-file-id">desktop-file id</ulink> in that case.
-		</para>
-		<para>
-			The <code>&lt;launchable/&gt;</code> tag is allowed to appear multiple times in the metainfo data. In case a software component has multiple launchable entries,
-			the software center might display a dialog to choose which entry to launch. If possible though, it should be avoided to add multiple <literal>launchable</literal> tags.
-		</para>
+		         This optional tag indicates possible methods to launch the software described in this component.
+                         It is allowed to appear multiple times in the metainfo data.
+		</para>
+
+		<para>
+		  The <code>&lt;launchable/&gt;</code> tag has a essential <literal>type</literal> property indicating the system that is used to launch the component.  The following types are allowed:
+                </para>
+                <variablelist>
+                  <varlistentry>
+                    <term>desktop-id</term>
+                    <listitem>
+                      <para>
+                        The application can be launched via a desktop file. The value of the tag is a
+			<ulink url="https://standards.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#desktop-file-id">desktop-file id</ulink>.
+                      </para>
+                      <para>
+                        In case a software component has multiple launchable entries,
+			the software center might display a dialog to choose which entry to launch.
+                        If possible though, it should be avoided to add multiple <literal>launchable</literal> tags of type <literal>desktop-id</literal>.
+                      </para>
+                    </listitem>
+                  </varlistentry>
+                  <varlistentry>
+                    <term>service</term>
+                    <listitem>
+                      <para>
+                        The software can be started, stopped, and monitored by the OS "init"
+                        facility, such as systemd. The value of the tag is a name that can be
+                        used with that facility, such as a systemd unit name.
+                      </para>
+                      <para>
+                        If more than one <literal>launchable</literal> tag of type <literal>service</literal> is
+                        present, a UI is expected to let the user manage all these services.  Thus multiple
```

Comment:
I usually try to be precise here, because in case someone wants to implement a UI for a new software center but doesn't have the specific knowledge about services, it's helpful to know what the spec expects from UIs.
It also helps in case someone looks at the spec at a later date and wonders what the intentions were for a specific feature.

Arguably this is a rather nit-picky case here, as in case of Unix services it's well established what you can do with them.